src/js/shared/countdown.js

```diff
@@ -76,8 +76,8 @@ function padZero(num) {
   return chars.map((c) => `<span class="countdown-digit">${c}</span>`).join('');
 }
 
-Countdown.hackathonStart = new Date(Date.UTC(2017, 0, 28, 13, 0, 0));
-Countdown.hackathonEnd = new Date(Date.UTC(2017, 0, 29, 12, 45, 0));
+Countdown.hackathonStart = new Date(Date.UTC(2018, 0, 20, 13, 0, 0));
```

Comment:
It would be better to get this information from dates.js — if you don't have time to do it now, could you add an issue?

Reply:
Yep, will do!

Reply:
Think we should remove this as irrelevant to the pull

Reply:
@moosichu

Reply:
#322